sources/appendix-value-extraction.asciidoc

```diff
@@ -280,30 +281,38 @@ JavaFX's property hierarchy falls into that category, as it defines specific `Pr
 @Email StringProperty emailProperty;
 ----
 
-The `ConstraintAppliesTo` annotation can be used to control the target of validation in such cases:
+Two defined sub-types of `javax.validation.Payload` can be used to control the target of validation in such cases via the constraint's `payload()` attribute:
 
-[source,java]
+.`Payload` types for unwrapping control
+====
+
+[source, JAVA, indent=0]
 ----
-@Target({ METHOD, FIELD, CONSTRUCTOR, PARAMETER })
-@Retention(RUNTIME)
-public @interface ConstraintsApplyTo {
-    ValidationTarget value();
-}
+include::{validation-api-source-dir}javax/validation/valueextraction/Unwrapping.java[lines=7..8;11..-1]
 ----
 
+====
+
 [source,java]
 ----
-public enum ValidationTarget {
-    ANNOTATED_ELEMENT,
-    WRAPPED_VALUE;
-}
+@Email(payload=Unwrapping.Unwrap.class) StringProperty emailProperty;
```

Comment:
Very minor gripe, but some padding around the = would be nice to be consistent with the rest.

This is a bad example as `StringProperty` will use the automatic unwrapping. Maybe, simply call it `StringWrapper` (it is used in an example below) or something similar?